pkg/sdn/plugin/proxy.go

```diff
@@ -50,15 +58,61 @@ func (proxy *ovsProxyPlugin) Start(baseHandler pconfig.EndpointsConfigHandler) e
 		return err
 	}
 
+	policies, err := proxy.registry.GetEgressNetworkPolicies()
+	if err != nil {
+		return fmt.Errorf("Could not get EgressNetworkPolicies: %s", err)
+	}
+	for _, policy := range policies {
+		proxy.updateNetworkPolicy(policy)
```

Comment:
There could be multiple policies on the same namespace. We need to consolidate all the policies for the namespace before assigning it to proxy.firewall[namespace]?
https://github.com/openshift/origin/pull/9982/commits/799ce2bc0da1917478989eea1a9236101cbaa73d#diff-39f5aa4f9b871145d31267e2f480b6adR110  will override existing policy for the namespace.